integration-cli/docker_api_containers_test.go

```diff
@@ -553,3 +553,43 @@ func TestPostContainerBindNormalVolume(t *testing.T) {
 
 	logDone("container REST API - can use path from normal volume as bind-mount to overwrite another volume")
 }
+
+func TestContainerApiPause(t *testing.T) {
+	defer deleteAllContainers()
+	runCmd := exec.Command(dockerBinary, "run", "-d", "busybox", "sleep", "30")
+	out, _, err := runCommandWithOutput(runCmd)
+
+	if err != nil {
+		t.Fatalf("failed to create a container: %s, %v", out, err)
+	}
+	ContainerID := stripTrailingCharacters(out)
+
+	if _, err = sockRequest("POST", "/containers/"+ContainerID+"/pause", nil); err != nil && !strings.Contains(err.Error(), "204 No Content") {
+		t.Fatalf("POST a container pause: sockRequest failed: %v", err)
+	}
+
+	pausedContainers, err := getSliceOfPausedContainers()
+
+	if err != nil {
+		t.Fatalf("error thrown while checking if containers were paused: %v", err)
+	}
+	if len(pausedContainers) != 1 || pausedContainers[0] == "" {
```

Comment:
right but what if there is a value but its not the ID of our container?
I just though it should be:   `... || pausedContainer[0] != ContainerID`